src/schema/comments.ts

```diff
@@ -98,7 +99,14 @@ export interface GQLUserComment {
   votedAt: Date | null;
 }
 
+export enum SortCommentsBy {
+  NewestFirst = 'newest',
+  OldestFirst = 'oldest',
+}
+
 export const typeDefs = /* GraphQL */ `
+  ${toGQLEnum(SortCommentsBy, 'SortCommentsBy')}
```

Comment:
@sshanzel you probably have circual dependency

Reply:
Yeah, was thinking that too, but wanted to confirm. Let me check this one quickly 👀

Reply:
fix it by importing from `import { toGQLEnum } from '../common/utils'`

Reply:
Basically try to avoid barrel files

Reply:
Yikes! Never knew doing that would be such an issue. Worked now. Thanks!